src/server/serializer.ts

```diff
@@ -108,6 +116,11 @@ export function serialize(data: unknown): SerializeResult {
           referenceArr.push(currentPath);
         }
         return 0;
+      } else if (isVNode(value)) {
+        requiresDeserializer = true;
+        const res = { __slot: key };
```

Comment:
This `__slot` isn't used for anything right?

How about `null`, `0`, or an empty object literal instead?

Reply:
Good point, that can be dropped. It's a leftover from an earlier iteration when I used this to infer the slot name. It's not needed anymore.